### PR TITLE
Create python script to run Lighthouse and send metrics to DataDog

### PR DESCRIPTION
**Pull request by ardelato**
## Description

This is a similar approach to the DataDog Lighthouse Integration seen here https://github.com/DataDog/integrations-extras/blob/master/lighthouse/datadog_checks/lighthouse/lighthouse.py. 

Most of the logic is in the `main.py` script which will iterate through a constant list of URLs, run Lighthouse on each URL, parse the results for core metrics, and send these metrics over to DataDog using the wrapper class.

In future pulls we attempt to run this as a cron task to constantly audit the list of URLs and send metrics to DataDog.

### CR

With this approach, we will be able to retrieve the network request size which will help to address the original issue described in https://github.com/iFixit/ifixit/discussions/47449.

At the moment, I decided to only send the cumulative layout shift, largest contentful paint, total blocking time, and page size. The first three metrics are weighted the most by Google. 

### QA

1. Run `pip install -r /path/to/requirements.txt` to install the required dependencies
2. Have `lighthouse` installed locally
3. Run `python3 lib/main.py` and confirm that lighthouse was run for each URL and the metrics show up in DataDog

**cc:** @andyg0808 
Closes: https://github.com/iFixit/ifixit/issues/48149

### CODE REVIEW

**Comment by deltuh-vee** [comment]
QA 🎬 
Lighthouse metrics can show up in Datadog.

File: lib/datadog_api.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        datadog.api.Metric.send(metrics=metric)
+
+
+if __name__ == '__main__':
+    tags = {'url': 'https://www.ifixit.com', 'test': 'test'}
+    tags = [f'{k}:{v}' for k, v in tags.items()]
+
+    dd_client = DataDogApiClient()
+    dd_client.submit_metric('lighthouse.cumulative_layout_shift', 0.5, tags)
```
Comment (mlahargou) [inline lib/datadog_api.py:33]: What do these points represent?

**Review by mlahargou — APPROVED** [review]
CR 📱 

Checks out to me!

File: lib/datadog_api.py (hunk 1 above)
Comment (ardelato) [reply lib/datadog_api.py:33]: They are just test points. This code block is under the `if __name__ == __main__` so if we run this file directly with `python lib/datadog_api.py` then it'll execute it.

**Pull request merged**